apps/tenant-management-webapp-e2e/src/integration/directory-service.feature

```diff
@@ -142,4 +142,48 @@ Feature: Directory-service
+    When the user clicks "eye" icon of "urn:ads:autotest:autotest-resource-types:api", "auto-test", "^Child Service .+" on resource types page
+    Then the user views "autotest-namepath", "Autotest:autotest-eventDefinition" in the details view of the resource type of "urn:ads:autotest:autotest-resource-types:api", "auto-test", "^Child Service .+" on resource types page
+    # Edit a resource type
+    When the user clicks "edit" icon of "urn:ads:autotest:autotest-resource-types:api", "auto-test", "^Child Service .+" on resource types page
+    Then the user "views" Edit resource type modal
+    And the user views Api dropdown is disabled in resource type modal
+    When the user enters "N/A", "auto-test-new", "^Land Title .+", "autotest-namepath-new", "autotest-DO-NOT-DELETE" in resource type modal
+    And the user clicks Cancel button in resource type modal
+    Then the user "should not view" the entry of "urn:ads:autotest:autotest-resource-types:api", "auto-test-new", "^Land Title .+" on resource types page
+    When the user clicks "edit" icon of "urn:ads:autotest:autotest-resource-types:api", "auto-test", "^Child Service .+" on resource types page
+    And the user enters "N/A", "auto-test-new", "^Land Title .+", "autotest-namepath-new", "autotest-DO-NOT-DELETE" in resource type modal
+    And the user clicks Save button in resource type modal
+    Then the user "views" the entry of "urn:ads:autotest:autotest-resource-types:api", "auto-test-new", "^Land Title .+" on resource types page
+    When the user clicks "eye" icon of "urn:ads:autotest:autotest-resource-types:api", "auto-test-new", "^Land Title .+" on resource types page
+    Then the user views "autotest-namepath-new", "autotest-DO-NOT-DELETE" in the details view of the resource type of "urn:ads:autotest:autotest-resource-types:api", "auto-test-new", "^Land Title .+" on resource types page
+    When the user clicks "edit" icon of "urn:ads:autotest:autotest-resource-types:api", "auto-test-new", "^Land Title .+" on resource types page
+    Then the user "views" Edit resource type modal
+    # Delete a resource type
+    When the user clicks "delete" icon of "urn:ads:autotest:autotest-resource-types:api", "auto-test-new", "^Land Title .+" on resource types page
+    Then the user views delete "resource type" confirmation modal for "auto-test-new"
+    When the user clicks Cancel button in delete confirmation modal
+    Then the user "views" the entry of "urn:ads:autotest:autotest-resource-types:api", "auto-test-new", "^Land Title .+" on resource types page
+    When the user clicks "delete" icon of "urn:ads:autotest:autotest-resource-types:api", "auto-test-new", "^Land Title .+" on resource types page
+    When the user clicks Delete button in delete confirmation modal
+    Then the user "should not view" the entry of "urn:ads:autotest:autotest-resource-types:api", "auto-test-new", "^Land Title .+" on resource types page
```

Comment:
And

Reply:
No need to change as the previous one has been changed to And.